examples/simulation/Tutorial2/macros/CMakeLists.txt

```
@@ -12,15 +12,25 @@ GENERATE_ROOT_TEST_SCRIPT(${CMAKE_CURRENT_SOURCE_DIR}/read_digis.C)
 GENERATE_ROOT_TEST_SCRIPT(${CMAKE_CURRENT_SOURCE_DIR}/run_background.C)
 GENERATE_ROOT_TEST_SCRIPT(${CMAKE_CURRENT_SOURCE_DIR}/run_signal.C)
 GENERATE_ROOT_TEST_SCRIPT(${CMAKE_CURRENT_SOURCE_DIR}/run_tutorial2.C)
+GENERATE_ROOT_TEST_SCRIPT(${CMAKE_CURRENT_SOURCE_DIR}/compare_seed_value.C)
 
 set(maxTestTime 60)
 
 add_test(NAME ex_sim_tutorial2
-         COMMAND ${CMAKE_CURRENT_BINARY_DIR}/run_tutorial2.sh)
+         COMMAND ${CMAKE_CURRENT_BINARY_DIR}/run_tutorial2.sh  10 \"TGeant4\" true 333)
 set_tests_properties(ex_sim_tutorial2 PROPERTIES
   TIMEOUT ${maxTestTime}
   PASS_REGULAR_EXPRESSION "Macro finished successfully"
   FIXTURES_SETUP fixtures.ex_sim_tutorial2_sim
+  RESOURCE_LOCK sim_paramFile
+)
+
+add_test(NAME ex_compare_seed_value
```

```suggestion
add_test(NAME ex_sim_tutorial2_compare_seed_value
```

For consistency, all tests in this directory have the prefix `ex_sim_tutorial2`, thx!

---

Done.

---

```
CMake Error at examples/simulation/Tutorial2/macros/CMakeLists.txt:30 (set_tests_properties):
  set_tests_properties Can not find test to add properties to:
  ex_sim_tutorial2_compare_seed_value

```

---

tbh, I would write
```cmake
set(test ex_sim_tutorial2_compare_seed_value)
add_test(NAME ${test} ...)
set_tests_properties(${test} ...)
```
to not have to change the same name in multiple places.

---

> ```
> CMake Error at examples/simulation/Tutorial2/macros/CMakeLists.txt:30 (set_tests_properties):
>   set_tests_properties Can not find test to add properties to:
>   ex_sim_tutorial2_compare_seed_value
> ```

Commit before thinking. I did not even test locally and messed up.

---

I agree. This is what I normally do, but I didn't want to change it in the existing CMakeLists.txt

---

BTW: Please update the copyright of the CMakeLists.txt as well.